vispy/visuals/rectangle.py

```diff
@@ -181,7 +181,7 @@ def radius(self, radius):
                                  " (got %s of length %d)" % (type(radius),
                                                              len(radius)))
 
-            if (radius > np.full(4, hw)).all():
+            if (radius > np.ones(4) * hw).all():
```

Comment:
Why not just `radius > hw`? It should be broadcast to the right shape automatically.